.github/workflows/update.yml

```diff
@@ -0,0 +1,25 @@
+name: update-dependencies
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened] # kittu remove this in final PR
+  schedule:
+    - cron: '0 4 * * 6'  # Every Saturday at 4 AM US time (America/New_York)
+
+jobs:
+  update-refs:
```

Comment:
I think we should update all refs in one step. We can't be sure of hte compatibility between hashes otherwise.